op-e2e/actions/helpers/l1_miner.go

```diff
@@ -196,6 +198,15 @@ func (s *L1Miner) IncludeTx(t Testing, tx *types.Transaction) *types.Receipt {
 	if tx.Type() == types.BlobTxType {
 		require.True(t, s.l1Cfg.Config.IsCancun(s.l1BuildingHeader.Number, s.l1BuildingHeader.Time), "L1 must be cancun to process blob tx")
 		sidecar := tx.BlobTxSidecar()
+		require.NotNil(t, sidecar, "missing sidecar in blob transaction")
+		hashes := tx.BlobHashes()
+		require.Greater(t, len(hashes), 0, "blobless blob transaction")
+		require.NoError(t, sidecar.ValidateBlobCommitmentHashes(hashes))
+		if s.l1Cfg.Config.IsOsaka(s.l1BuildingHeader.Number, s.l1BuildingHeader.Time) {
+			require.NoError(t, validateBlobSidecarOsaka(sidecar, hashes))
+		} else {
+			require.NoError(t, validateBlobSidecarLegacy(sidecar, hashes))
+		}
 		if sidecar != nil {
 			s.l1BuildingBlobSidecars = append(s.l1BuildingBlobSidecars, sidecar)
 		}
```

Comment:
Note that you already require above that it is now not nil, so you can remove this check now. Wondering if there ever was a test without a sidecar?